src/app/client/src/app/modules/certificate/components/create-template/create-template.component.ts

```diff
@@ -352,4 +367,75 @@ urltoFile(url, filename, mimeType) {
       this.SECOND_PANEL_LAYOUT = this.layoutService.redoLayoutCSS(1, null, COLUMN_TYPE.fullLayout);
     }
   }
+
+  elementClicked(e: any) {
+    this.selectedSVGObject = {
+      type: _.get(e, 'type'),
+      value: _.get(e, 'element.textContent'),
+      element: e.element
+    };
+    if (e.type === 'image') {
+      this.logoType = {type: 'LOGO', index: 0,  key:'LOGO1'};
+      this.browseImage.getAssetList();
+    }
+    
+    this.showSVGInputModal = true;
+  }
+  updateSVGInputTag() {
+    this.showSVGInputModal = false;
+    this.edit.next({
+      element: this.selectedSVGObject.element,
+      type: 'text',
+      value: this.selectedSVGObject.value
+    });
+    this.selectedSVGObject = {};
+  }
+  closeSVGInputModal() {
+    this.showSVGInputModal = false;
+  }
+
+  svgAssetData(imageObj) {
+    this.getBase64FromUrl(_.get(imageObj, 'url')).then((base64String: string) => {
+      this.showSVGInputModal = false;
+      this.edit.next({
+        element: this.selectedSVGObject.element,
+        type: 'image',
+        value: base64String
+      });
+      this.selectedSVGObject = {};
+    });
+  }
+  getBase64FromUrl = async (url) => {
+    const data = await fetch(url);
+    const blob = await data.blob();
+    return new Promise((resolve) => {
+      const reader = new FileReader();
+      reader.readAsDataURL(blob);
+      reader.onloadend = () => {
+        const base64data = reader.result;
+        resolve(base64data);
+      }
+    });
+  }
+  toggleSVGPreview() {
+    this.previewButton = this.previewButton == 'show' ? 'hide' : 'show';
+    this.togglePreview.next(this.previewButton);
+  }
+
+  saveUpdatedCertificate() {
+    this.save.next('');
+    this.previewUpdatedSVGCertificate();
+    setTimeout(() => {
+      const channel = this.userService.channel;
+      const request = this.certConfigModalInstance.prepareCreateAssetRequest(_.get(this.createTemplateForm, 'value'), channel, this.selectedCertificate, this.images);
+      request.request.asset.code = 'code name';
```

Comment:
@rajeevsathish @yravinderkumar33 
Intentionally hard coded; will be reverted in next commit due to discussion on hold.